pulp_ansible/app/viewsets.py

```diff
@@ -271,6 +279,54 @@ class Meta:
         }
 
 
+class SigstoreSignatureFilter(ContentFilter):
+    """
+    A filter for Sigstore signatures.
+    """
+
+    signed_collection = HyperlinkRelatedFilter(
+        field_name="signed_collection", help_text=_("Filter signatures for collection version")
+    )
+    sigstore_signing_service = HyperlinkRelatedFilter(
+        field_name="sigstore_signing_service",
+        help_text=_("Filter Sigstore signatures produced by Sigstore Signing Service"),
+    )
+
+    class Meta:
+        model = CollectionVersionSigstoreSignature
+        fields = {
+            "signed_collection": ["exact"],
+            "sigstore_bundle_sha256_hash": ["exact", "in"],
+            "sigstore_signing_service": ["exact"],
+        }
+
+
+class SigstoreSigningServiceViewSet(
+    NamedModelViewSet, mixins.RetrieveModelMixin, mixins.ListModelMixin, RolesMixin
```

Comment:
```suggestion
    NamedModelViewSet, mixins.RetrieveModelMixin, mixins.ListModelMixin
```

Reply:
Regarding `ansible-sign`, I am still waiting for the sigstore code to be merged there (see https://github.com/ansible/ansible-sign/pull/46), but the code should indeed be replaced when this is done. Let me get back to the `ansible-sign` maintainers and I will replace it.

Reply:
The verification service would still be required as policies for verification must be specified with sigstore (i.e. signer's identity, OIDC issuer, etc).

Reply:
I think, I said this before. There is no need for a verification service.
The design of the signing service is to protect secrets. Verification must always work without secrets. Otherwise your cryptosystem is broken. No secrets means: Everything lives in the database and is configurable via the api on the very Repository or Remote supposed to protect itself from improperly signed content.

Reply:
There is no need for any secret here, the parameters specified are public and are simple text fields used by the sigstore client to verify the signature, exactly like you would use a public GPG key.

Reply:
I think the term "verification service" might be misused here. It is actually more of a static configuration than a "service".